app/src/main-process/ordered-webrequest.ts

```diff
@@ -140,7 +140,7 @@ export class OrderedWebRequest {
     this.onBeforeRequest = new AsyncListenerSet(
       webRequest.onBeforeRequest.bind(webRequest),
       async (listeners, details) => {
-        let response: Response = {}
+        let response: CallbackResponse = {}
```

Comment:
Here is new interface:

![image](https://user-images.githubusercontent.com/75402236/209684898-6eda7fda-cf7d-46bf-b439-39dc5bab19ef.png)


If you look down at line 51, `cancel` and `redirectURL` are the expected properties.